packages/dbadmin/src/tools/init-tables-sql.ts

```diff
@@ -106,6 +107,20 @@ CREATE TABLE IF NOT EXISTS REPO_VERSIONS (
 
 INSERT INTO REPO_VERSIONS 
     VALUES (0, NOW(), 'repository_id');
+
+-- this table contains "global variables per transaction"
+CREATE TABLE IF NOT EXISTS CURRENT_DATA (
```

Comment:
So, the `CURRENT_DATA` table is really a key-value store?
All these `SELECT value INTO repo_version FROM CURRENT_DATA WHERE key = 'repo.version'` look a bit...“untyped”.

Reply:
Current data is a table of global variables, so indeed a key-value store.
It is untyped, the alternative is to have a key-value store for each type, maybe that is a better approach.

Reply:
Maybe the key can be typed as an enum? That way there's at least some type safety.

Reply:
What do you mean "type key as enum"?

Reply:
I mean using an enum type as the type for the first column: https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
Ok, but the problem is that the second column is untyped and needs to be converted to the correct type.

Reply:
What about a table with just one column, instead?

Reply:
That is an option, with basically the same type-safety as a key-value table per value type.

Reply:
Kept it for now, as this is not used at many places.